### PR TITLE
Adding wrapper for BCFtools norm.

### PR DESCRIPTION
**Pull request by heuermh**

### CODE REVIEW

File: cli/src/main/scala/org/bdgenomics/cannoli/cli/Bcftools.scala [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  def run(sc: SparkContext) {
+    val input: VariantContextRDD = sc.loadVcf(args.inputPath, stringency = stringency)
+
+    implicit val tFormatter = VCFInFormatter
+    implicit val uFormatter = new VCFOutFormatter
+
+    val bcftoolsCommand = if (args.useDocker) {
+      Seq("docker",
+        "run",
+        "--rm",
```
Comment (fnothaft) [inline cli/src/main/scala/org/bdgenomics/cannoli/cli/Bcftools.scala]: You'll need `-it` as well.

Comment (heuermh) [reply cli/src/main/scala/org/bdgenomics/cannoli/cli/Bcftools.scala]: There is no such option in the latest version of docker, I assume you mean `--interactive`?
```
$ docker --version
Docker version 17.09.0-ce, build afdb6d4
```

Comment (fnothaft) [reply cli/src/main/scala/org/bdgenomics/cannoli/cli/Bcftools.scala]: Yes, sorry about that!

**Comment by heuermh** [comment]
With an additional `--mount` argument for the reference argument, this works for me locally using docker.

**Pull request merged**